gui/default/syncthing/core/syncthingController.js

```diff
@@ -1270,6 +1290,10 @@ angular.module('syncthing.core')
             }
         }
 
+        $scope.globalDiskChanges = function () {
+            $('#globalDiskChanges').modal();
+        }
```

Comment:
I know the above line is already inconsistent (looking at the above function), but can we have a `;` at the end of the function declaration.